tools-v2/pkg/cli/command/curvebs/check/server/get_copyset_ids.go

```diff
@@ -0,0 +1,145 @@
+/*
+ *  Copyright (c) 2022 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: CurveCli
+ * Created Date: 2023-05-07
+ * Author: pengpengSir
+ */
+package server
+
+import (
+	"context"
+	"fmt"
+
+	cmderror "github.com/opencurve/curve/tools-v2/internal/error"
+	basecmd "github.com/opencurve/curve/tools-v2/pkg/cli/command"
+	"github.com/opencurve/curve/tools-v2/pkg/config"
+	"github.com/opencurve/curve/tools-v2/pkg/output"
+	"github.com/opencurve/curve/tools-v2/proto/proto/topology"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+)
+
+type GetChunkServerCopysetsRpc struct {
+	Info           *basecmd.Rpc
+	Request        *topology.GetCopySetsInChunkServerRequest
+	topologyClient topology.TopologyServiceClient
+}
+
+type CopysetidsCommand struct {
+	basecmd.FinalCurveCmd
+	Rpc      *GetChunkServerCopysetsRpc
+	Response *topology.GetCopySetsInChunkServerResponse
+}
+
+var _ basecmd.FinalCurveCmdFunc = (*CopysetidsCommand)(nil)
+
+func NewCopysetCommand() *cobra.Command {
+	return NewCopysetidsCommand().Cmd
+}
+
+func NewCopysetidsCommand() *CopysetidsCommand {
+	ccCmd := &CopysetidsCommand{
+		FinalCurveCmd: basecmd.FinalCurveCmd{},
+	}
+	basecmd.NewFinalCurveCli(&ccCmd.FinalCurveCmd, ccCmd)
+	return ccCmd
+}
+
+func (tRpc *GetChunkServerCopysetsRpc) NewRpcClient(cc grpc.ClientConnInterface) {
+	tRpc.topologyClient = topology.NewTopologyServiceClient(cc)
+}
+
+func (tRpc *GetChunkServerCopysetsRpc) Stub_Func(ctx context.Context) (interface{}, error) {
+	return tRpc.topologyClient.GetCopySetsInChunkServer(ctx, tRpc.Request)
+}
+
+func (cCmd *CopysetidsCommand) AddFlags() {
+	config.AddBsMdsFlagOption(cCmd.Cmd)
+	config.AddRpcRetryTimesFlag(cCmd.Cmd)
+	config.AddRpcTimeoutFlag(cCmd.Cmd)
+	config.AddBsChunkServerIDOptionFlag(cCmd.Cmd)
+	config.AddBsHostIpFlag(cCmd.Cmd)
```

Comment:
Both hostip and serverip are essentially ip, can these two be called ip? Is it convenient to pass it on?